platform/lib/pipeline/markdownDocument.js

```diff
@@ -181,6 +182,16 @@ class MarkdownDocument {
     return contents;
   }
 
+  /**
+   * Rewrite relative links and append the given base path to them
+   * @param  {String} contents
+   * @return {String}          The rewritten content
+   */
+  rewriteRelativePaths(base) {
+    this._contents = this._contents.replace(/\[([^\]]+)\]\((?!http|#)([^\)]+)\)/g,
```

Comment:
I'd be feeling better doing this via a markdown parser and `new URL(string, base)`.